src/main/java/com/epam/restbookservice/controllers/BookController.java

```diff
@@ -0,0 +1,47 @@
+package com.epam.restbookservice.controllers;
+
+import com.epam.restbookservice.domain.Book;
+import com.epam.restbookservice.services.BookService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+
+@RestController
+@RequestMapping("/books")
+@RequiredArgsConstructor
+public class BookController {
+
+    @Autowired
+    BookService bookService;
+
+    @PostMapping(consumes = "application/json")
```

Comment:
I guess `"application/json"` is not needed since `@RestController` should take care of it but I may be wrong